src/pages/events/weekender-2025-11/ArtistsPage.jsx

```
@@ -0,0 +1,187 @@
+                                        >
+                                            {featuredArtist.specialties.map((specialty, index) => (
+                                                <li
+                                                    key={index}
+                                                    className="font-mono"
+                                                    style={{
+                                                        fontSize: 'var(--font-size-sm)',
+                                                        padding: 'var(--space-xs) 0',
+                                                        color: 'var(--color-gray-600)',
+                                                    }}
+                                                >
+                                                    // {specialty}
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)